unit-3-js-labs/shape-functions/index.js

```diff
@@ -0,0 +1,28 @@
+function getAreaOfCircle(radius) {
+    return Math.PI * radius**2
```

Comment:
Use spaces around the exponent operator

```suggestion
    return Math.PI * radius ** 2
```